rules/forbidden_actions.tex

```diff
@@ -107,12 +107,12 @@ \subsection{Forbidden Motion}
 ``Forbidden Motion'' penalties do not follow the standard removal procedure, and hence do not count towards the incremental penalty count.
 A robot will be moved back to its original position if it has moved significantly before becoming penalized.
 
-\subsubsection{Motion in Initial}
-\label{sec:motion_in_initial}
+\subsubsection{Motion in Visual}
```

Comment:
I believe the current name "Visual" for the state where robots are waiting for the referee's gesture is not very intuitive. I suggest we use **"Standby"** instead, as it clearly conveys a state of readiness while awaiting further instructions.

Reply:
I will make the update